pkg/oauthserver/authenticator/password/keystonepassword/keystonepassword.go

```diff
@@ -65,12 +65,10 @@ func (a keystonePasswordAuthenticator) AuthenticatePassword(username, password s
 	}
 
 	client.HTTPClient = *a.client
-	err = openstack.AuthenticateV3(client, opts)
+	err = openstack.AuthenticateV3(client, &opts, gophercloud.EndpointOpts{})
```

Comment:
Do any of the options need to make it into EndpointOpts, or is empty always appropriate?

Reply:
In our case this is fine. The endpoint opts are used to explicitly select an endpoint from the catalog, for example if there are several regions. In our case this is not required, because we have only one keystone endpoint.